src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,13 @@
 import React from "react";
 
-const NumberButton = () => {
+const NumberButton = (props) => {
+console.log("NumberButton", props)
   return (
     <>
       {/* Display a button element rendering the data being passed down from the parent container on props */}
+      <button id={ props.text <  1 ? 'zero' : null } className="number-button" onClick={() => props.addNumber(props.text)}>{props.text }</button>
```

Comment:
I think I know what you're doing here.  You're assigning the 0 button a classname you can target.  It might have been eaiser to use a pseudo selector and select a nth child?